src/visual.ts

```diff
@@ -668,7 +687,8 @@ module powerbi.extensibility.visual {
 
             // Draw Labels
             if (model.settings.labels.show) {
-                barSelection.enter().append("text").classed("title", true).attr({
+                let textElement = barSelection.enter().append("text");
+                textElement.classed("title", true).attr({
```

Comment:
Please note that `attr` should be called on `textElement`. You shouldn't move it.